assets/javascripts/discourse/components/modal/brightcove-upload.hbs

```diff
@@ -1,7 +1,8 @@
-<DModalBody
-  @title="brightcove.modal_title"
-  @subtitle="brightcove.modal_subtitle"
+<DModal
+  @title={{i18n "brightcove.modal_title"}}
+  @subtitle={{i18n "brightcove.modal_subtitle"}}
   class="brightcove-upload-modal"
+  @closeModal={{@closeModal}}
 >
   <h3>{{i18n "brightcove.file"}}</h3>
```

Comment:
btw, that stuff doesn't _need_ to be placed in a `<:body>` block?

Reply:
Correct. DModal will use the 'default block' as the body content if `<:body>` is not specified.

The only reason to use `<:body>` is if you also need other named blocks like `<:footer>`